tests/console/sshd.pm

```diff
@@ -70,7 +70,15 @@ sub run {
     }
 
     # Restart sshd and check it's status
-    systemctl 'restart sshd';
+    my $ret          = systemctl('restart sshd', ignore_failure => 1);
+    my $fips_enabled = script_output('cat /proc/sys/crypto/fips_enabled');
```

Comment:
Why not make this a boolean style variable directly here?

```suggestion
    my $fips_enabled = script_output('cat /proc/sys/crypto/fips_enabled') eq '1';
```

And then

```
if (($ret != 0) && ($fips_enabled)) {
```

Reply:
Much more elegant!